invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
+
+    on_fail = "Some of the packages need to be updated."
+    on_success = "All packages are up to date."
+
+    run_steps(steps, on_fail, on_success)
 
 
 @packages.command()
+@click.argument("packages", nargs=-1, type=str)
 @pass_cli_config
-def update(cli_config):
-    """Update a single Python python package."""
-    raise click.UsageError("Not supported yet.")
+def update(cli_config, packages=[]):
+    """Update all or some Python python packages."""
+    if len(packages) >= 1:
+        steps = PackagesCommands.update_packages(packages)
+        on_fail = f"Failed to update packages {packages}."
+        on_success = f"Packages {packages} installed successfully."
+    else:
+        steps = PackagesCommands.update_packages("")
```

Comment:
major: this is not a good practice... either:
- The parameter is *mandatory* as it is now, then you **MUST** pass something meaningfull (or `None`)... as commented [here](https://github.com/inveniosoftware/invenio-cli/pull/242#discussion_r629384174). Passing `""` is too hackish.
- The parameter is *optional*, you deal with it in the function itself as mentioned [here](https://github.com/inveniosoftware/invenio-cli/pull/242#discussion_r629382704). Then you can simply call the function (`PackagesCommands.update_packages()`)